### PR TITLE
Implement itemAnswerGet service

### PR DESCRIPTION
**Pull request by zenovich**
[itemAnswerGet](https://docs.google.com/document/d/15Xxmatrs_7my5ioriikL6YgOU4I0sAVcq5ySKinc0Nw/edit#bookmark=id.p0ptd8aoosrg)

### CODE REVIEW

File: app/database/db.go [hunk 1]
```diff
@@ -270,9 +270,13 @@ func (conn *DB) Scan(dest interface{}) *DB {
 }
 
 // ScanIntoSliceOfMaps scans value into a slice of maps
-func (conn *DB) ScanIntoSliceOfMaps(dest *[]map[string]interface{}) *DB {
+func (conn *DB) ScanIntoSliceOfMaps(dest *[]map[string]interface{}) *DB { // nolint:gocyclo
```
Comment (smadbe) [inline app/database/db.go]: It is cheating to hide an actual lint warning. Cannot you find a way to reduce the complexity of this function?

File: app/api/answers/get.feature [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    | 23 | Group C    | Class     |
+  And the database has the following table 'groups_groups':
+    | ID | idGroupParent | idGroupChild | sType              |
+    | 61 | 13            | 11           | invitationAccepted |
+    | 62 | 13            | 21           | requestAccepted    |
+  And the database has the following table 'groups_ancestors':
+    | ID | idGroupAncestor | idGroupChild | bIsSelf |
+    | 71 | 11              | 11           | 1       |
+    | 72 | 12              | 12           | 1       |
+    | 73 | 13              | 13           | 1       |
```
[the remark this answers is not in the document]
Comment (smadbe) [reply app/api/answers/get.feature]: You can ignore this :-)

File: app/database/db.go (hunk 1 above)
Comment (zenovich) [reply app/database/db.go]: fixed

File: app/api/answers/get.feature [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    | 200 | 0            |
+    | 210 | 1            |
+  And the database has the following table 'groups_items':
+    | ID | idGroup | idItem | sCachedFullAccessDate | sCachedPartialAccessDate |
+    | 43 | 13      | 200    | 2017-05-29T06:38:38Z  | 2017-05-29T06:38:38Z     |
+    | 46 | 23      | 210    | 2017-05-29T06:38:38Z  | 2017-05-29T06:38:38Z     |
+  And the database has the following table 'users_answers':
+    | ID | idUser | idItem | idAttempt | sType      | sState  | sAnswer  | sLangProg | sSubmissionDate     | iScore | bValidated | sGradingDate        | idUserGrader |
+    | 1  | 1      | 200    | 150       | Submission | Current | print(1) | python    | 2017-05-29 06:38:38 | 100    | true       | 2018-05-29 06:38:38 | 123          |
+    | 2  | 1      | 210    | 250       | Submission | Current | print(2) | python    | 2017-05-29 06:38:38 | 100    | true       | 2019-05-29 06:38:38 | 456          |
```
[the remark this answers is not in the document]
Comment (zenovich) [reply app/api/answers/get.feature]: fixed

**Pull request merged**